src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -183,7 +183,14 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
             if (searchParams.Include?.Count > 0)
             {
                 searchExpressions.AddRange(searchParams.Include.Select(
-                    q => _expressionParser.ParseInclude(parsedResourceType.ToString(), q))
+                    q => _expressionParser.ParseInclude(parsedResourceType.ToString(), q, false /* reversed */))
```

Comment:
possibly default the reversed argument to false?

Reply:
possible however I prefer stating it explicitly. the way I think about it is as a two different include types

Reply:
ok, as long as this is documented